src/main/java/org/openelisglobal/search/service/LuceneSearchResultsService.java

```diff
@@ -0,0 +1,35 @@
+package org.openelisglobal.search.service;
+
+import java.util.List;
+import javax.transaction.Transactional;
+import org.openelisglobal.common.provider.query.PatientSearchResults;
+import org.openelisglobal.sample.dao.SearchResultsDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LuceneSearchResultsService implements SearchResultsService {
```

Comment:
rename to `LuceneResultsServiceImpl`

Reply:
I renamed this service to `LuceneSearchResultsServiceImpl` to stay consistent with the other renamings you suggested. I have also renamed other search service and DAO implementations. Let me know if any other change is needed.
Thank you.

Reply:
Thanks @rahul6603